front50-web/src/test/groovy/com/netflix/spinnaker/front50/controllers/PipelineControllerSpec.groovy

```diff
@@ -253,6 +253,14 @@ class PipelineControllerSpec extends Specification {
         .collect(Collectors.toList())
     }
 
+    @Override
+    Pipeline getPipelineByName(String application, String pipelineName, boolean refresh) {
+      map.values().stream()
+        .filter({ p -> p.getApplication().equalsIgnoreCase(application) })
+        .filter({ p -> p.getName().equalsIgnoreCase(pipelineName) })
```

Comment:
YAY a case where we ignore case :)

Reply:
Trying to follow what similar methods do above ...